aat/sessionmeta_test.go

```diff
@@ -48,6 +48,16 @@ func TestMetaEventOnJoin(t *testing.T) {
 		t.Fatal("subscribe error:", err)
 	}
 
+	// Wait for any event from subscriber joining.
+	var timeout bool
+	for !timeout {
+		select {
+		case <-errChan:
+		case <-time.After(200 * time.Millisecond):
+			timeout = true
+		}
+	}
```

Comment:
This is used to catch and discard the event published when the subscriber joins the realm, if that event has not yet been published before the subscription is created.  The test discards this, if it happens, because it is not reliable.  The test is interested in checking for the event triggered by another session's subscription.